src/redux/raw/selectors.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Selectors for raw
+ */
+
+
+ export function getRawTests(state) {
+   if (state.raw.tests.data) {
+     return state.raw.tests.data;
+   }
+   return [];
+ }
```

Comment:
If you do switch it to not use combineReducers, you' need to update this to just read `raw.data` instead of `raw.tests.data`.